test/test_python_dispatch.py

```diff
@@ -803,6 +803,26 @@ def foo_meta(x, dim):
         self.assertEqual(result.shape, foo_meta(x, 1).shape)
         foo._destroy()
 
+    def test_duplicate_impl(self):
+        @custom_op('_torch_testing::foo')
+        def foo(x: torch.Tensor, dim: int) -> torch.Tensor:
+            ...
+
+        @foo.impl_abstract()
+        def foo_meta(x, dim):
+            output_shape = list(x.shape)
+            del output_shape[dim]
+            return x.new_empty(output_shape)
+
+        with self.assertRaisesRegex(RuntimeError, "already has a abstract impl"):
```

Comment:
nit: the error also includes the location of the registration, is it worth including that in the regex? (since grabbing it based on a hardcoded number of stack frames seems prone to drift over time).

Reply:
Yeah that's a good idea